apps/geo/filter_set.py

```diff
@@ -87,8 +87,13 @@ class Meta:
             },
         }
 
-
 # ------------------------------ Graphql filters -----------------------------------
+
+
+class Unaccent(models.Func):
+    function = 'UNACCENT'
+
```

Comment:
Not required. Django provides this by default.
https://docs.djangoproject.com/en/4.2/ref/contrib/postgres/lookups/#unaccent